src/CSF.SRDashboard.Client/PageValidators/SearchErrorObj.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.PageValidators
+{
+    public enum errorType
+    {
+        CRITERIA,
+        NO_RESULT
+    }
+    public class SearchErrorObj
+    {
+
+        public string message { get; set; }
```

Comment:
Capitalize the first word of a public parameter.
```suggestion
        public string Message { get; set; }
```